src/Controller/IncentiveImportController.php

```diff
@@ -16,9 +17,12 @@
 
 class IncentiveImportController extends BaseController
 {
-    public function __construct(EntityManagerInterface $em)
+    public function __construct(EntityManagerInterface $em, EnvironmentService $env)
     {
         parent::__construct($em);
+        if (!$env->isLocal()) {
```

Comment:
Should we use a feature flag configuration instead?

Reply:
I am not sure if we want to use a feature flag since this is not completely done?